tests/test_model.py

```diff
@@ -1,9 +1,12 @@
-
 import unittest
 import numpy as np
-
+import logging
 from lsst.ts.ataos import Model
 
+logging.basicConfig(level=logging.DEBUG)
```

Comment:
Don't set the log level in the module higher hierarchy. If you need debug level when running unit test you can do 

```
pytest --log-level DEBUG
```

Reply:
ok, will remove. The bad thing about that is that it puts EVERYTHING at a level of debug, so the verbosity can get pretty out of control pretty fast.